airbyte-webapp/src/packages/cloud/views/layout/SideBar/SideBar.module.scss

```
@@ -0,0 +1,43 @@
+@use "../../../../../scss/colors";
+
+.navBar {
+  z-index: 9999;
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%;
+  width: 100px;
+  min-width: 65px;
+  background-color: colors.$dark-blue;
+  padding: 23px 3px 15px 4px;
```

Paired with Nico yesterday on some things.  He's asked that these be centered
```suggestion
  padding: 23px 4px 15px 4px;
```

---

Fixed! Thanks